src/components/Nav.js

```diff
@@ -0,0 +1,37 @@
+import { Link } from "gatsby"
+import React from "react"
+import styled from "styled-components"
+
+if (typeof window !== "undefined") {
+  // eslint-disable-next-line global-require
+  require("smooth-scroll")('a[href*="#"]', { offset: 100 })
```

Comment:
there is an error when trying to run `yarn develop` because the `package smooth-scroll` isn't installed. Simple fix - `yarn add smooth-scroll`

Reply:
I commented it out to get it working, if we do actually want `smooth-scroll` then read 🔝